packages/@aws-cdk/aws-codebuild/lib/source.ts

```diff
@@ -668,6 +702,17 @@ class GitHubEnterpriseSource extends ThirdPartyGitSource {
       buildTriggers: superConfig.buildTriggers,
     };
   }
+
+  private hasCommitMessageFilterAndPREvent() {
```

Comment:
```suggestion
  private hasCommitMessageFilterAndPrEvent() {
```

Reply:
by the way, I had noticed that one cannot use `FILE_PATH` filters with GitHub Enterprise Server pull request events either. it may be out of scope for this ticket, but do we want to create a similar `hasFilePathFilterAndPrEvent` method to check for that?

Reply:
Yes, that would be awesome, thanks!